external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -117,12 +117,6 @@ fn unexpected_token_error_(
     ))
 }
 
-fn add_type_args_ambiguity_label(loc: Loc, mut diag: Box<Diagnostic>) -> Box<Diagnostic> {
```

Comment:
Why did this go away?

Reply:
It was no longer used after the "main" parser resilience PR (https://github.com/MystenLabs/sui/pull/16673) landed. This was due to the fact that generating this additional label was predicated on `parse_comma_list` returning an error which it no longer does. It just got more explicit in the current PR as `parse_optional_type_args` (wrapping `parse_comma_list`) now also no longer returns an error.